crates/nargo_cli/Cargo.toml

```diff
@@ -34,14 +34,14 @@ const_format = "0.2.30"
 hex = "0.4.2"
 serde_json = "1.0"
 termcolor = "1.1.2"
-tempdir = "0.3.7"
 color-eyre = "0.6.2"
 
 # Backends
 aztec_backend = { optional = true, package = "barretenberg_static_lib", git = "https://github.com/noir-lang/aztec_backend", rev = "26178359a2251e885f15f0a4d1a686afda04aec9" }
 aztec_wasm_backend = { optional = true, package = "barretenberg_wasm", git = "https://github.com/noir-lang/aztec_backend", rev = "26178359a2251e885f15f0a4d1a686afda04aec9" }
 
 [dev-dependencies]
+tempdir = "0.3.7"
```

Comment:
If tempdir is only used for these integration tests, can we drop it for `CARGO_TARGET_TMPDIR` as defined by https://doc.rust-lang.org/cargo/reference/environment-variables.html 

> Only set when building [integration test](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#integration-tests) or benchmark code. This is a path to a directory inside the target directory where integration tests or benchmarks are free to put any data needed by the tests/benches. Cargo initially creates this directory but doesn't manage its content in any way, this is the responsibility of the test code.

I wanted to look into this after we landed Nix stuff, as I'm not sure what it is set to for the nix sandbox.

Reply:
oooh, that's nice to know about. Not sure how Nix interacts with this but we can handle that later.

Reply:
Actually, this has a couple of drawbacks compared to tempdir in that we don't get any of the cleanup behaviour. We'd then have to implement similar logic to tempdir to ensure that we keep this directory clean.